src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Controllers/ApprenticeController.cs

```diff
@@ -130,5 +122,32 @@ public async Task<IActionResult> SelectEmployer(SelectEmployerRequest request)
 
             return View(viewModel);
         }
+
+        [HttpGet]
+        [Route("{apprenticeshipHashedId}/change-employer/price", Name = RouteNames.ApprenticePrice)]
+        [DasAuthorize(CommitmentOperation.AccessApprenticeship, ProviderFeature.ChangeOfEmployer)]
+        public async Task<IActionResult> Price(PriceRequest request)
+        {
+            var model = await _modelMapper.Map<PriceViewModel>(request);
+            return View(model);
+        }
+
+
+        [HttpPost]
+        [Route("{apprenticeshipHashedId}/change-employer/price", Name = RouteNames.ApprenticePrice)]
+        [DasAuthorize(CommitmentOperation.AccessApprenticeship, ProviderFeature.ChangeOfEmployer)]
+        public async Task<IActionResult> Price(PriceViewModel viewModel)
+        {
+            var request = await _modelMapper.Map<ChangeOfEmployerRequest>(viewModel);
+            return RedirectToRoute(RouteNames.ApprenticeConfirmChangeOfEmployer, request);
+        }
+
+        [HttpGet]
+        [Route("{apprenticeshipHashedId}/change-employer/confirm", Name = RouteNames.ApprenticeConfirmChangeOfEmployer)]
+        [DasAuthorize(CommitmentOperation.AccessApprenticeship, ProviderFeature.ChangeOfEmployer)]
+        public IActionResult ConfirmChangeOfEmployer(ChangeOfEmployerRequest request)
```

Comment:
Trivial, but this action should just be called `Confirm` and the request `ConfirmRequest`.